### ISSUE TITLE
-cli-run-interval can be specified without -cli

### ISSUE THREAD

**Issue by ahmetb** [GoogleCloudPlatform/cloud-run-release-manager#91]
```
go run ./cmd/operator -min-requests 0 -min-wait 0 -cli-run-interval 10s
INFO[0000] -project not specified, trying to autodetect one
INFO[0000] project detected: ahmetb-demo
INFO[0000] starting server                               addr=":8080"
```

**Comment by gvso** [comment]
Yeah, this doesn't sound good. I wonder how we can check for this since there's a default value for `-cli-run-interval`, so a simple `if !flCLI && flCLILoopInterval > 0` is not sufficient.

**Comment by ahmetb** [comment]
> Yeah, this doesn't sound good. I wonder how we can check for this since there's a default value for -cli-run-interval, so a simple if !flCLI && flCLILoopInterval > 0 is not sufficient.


yeah that's valid, maybe we don't do anything for now.
in the future we can require a non-0 value to be explicitly specified (default≤0). that way we can force users to use those two flags together.
